packages/account/src/Helpers/utils.ts

```diff
@@ -1,8 +1,16 @@
+import React from 'react';
 import { filterObjProperties, toMoment, validLength, validName, getIDVNotApplicableOption } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import { ResidenceList, GetSettings, GetAccountStatus } from '@deriv/api-types';
 import { FormikValues } from 'formik';
 import { getIDVDocumentConfig } from '../Constants/idv-document-config';
+import { TServerError } from '../Types/common.type';
+
+declare global {
+    interface Window {
+        clipboardData: DataTransfer;
+    }
+}
```

Comment:
why? 👀

Reply:
we need to extend types for avoiding TS error on line 77 (window.clipboardData)

Reply:
@yauheni-deriv yes, that's right but @shahzaib-deriv made a good point, `types/global.d.ts` is a better place to put it:
<img width="774" alt="Screenshot 2023-08-23 at 5 16 50 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/e133e7ff-b931-4e5a-9bcd-1f3f22b03321">
:)

Reply:
well, I intended to do it. 
let me try to refactor. thanks guys 👍